tests/test_job_management.py

```diff
@@ -1404,14 +1408,30 @@ def test_email_scws_list(dispatcher_long_living_fixture,
         elif scw_list_format == 'string':
             params['scw_list'] = scw_list_string
 
-    jdata = ask(server,
-                params,
-                method=ask_method,
-                max_time_s=150,
-                expected_query_status=None,
-                expected_status_code=None,
-                files=scw_list_file_obj
-                )
+    # this sets global variable
+    requests.get(server + '/api/par-names')
```

Comment:
But by clearing up and re-initializing the `params_not_to_be_included` as it is done now, this should no longer be necessary...?

Reply:
This is for test, and this triggers the problem when `params_not_to_be_included` is not cleared up. So by keeping it in the test, we make sure that this particular problem is actually fixed.